cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -298,24 +291,6 @@ private CristinSubjectField extractSubjectField() {
         return subjectField;
     }
 
-    private CristinJournalPublication extractCristinJournalPublication() {
```

Comment:
Moved logic for building a confirmed or unconfirmed Journal in a separate class.